nautobot_ssot/templates/nautobot_ssot/sync_detail.html

```diff
@@ -119,6 +119,41 @@
                         </td>
                     </tr>
                 </table>
+                <div class="panel-heading">
+                    <strong>Performance stats</strong>
+                </div>
+                <table class="table table-hover panel-body attr-table">
+                    <tr>
+                        <td>Time loading Source</td>
+                        <td>{{ object.load_time_source | shorter_timedelta }}</td>
+                    </tr>
+                    <tr>
+                        <td>Time loading Target</td>
+                        <td>{{ object.load_time_target | shorter_timedelta }}</td>
+                    </tr>
+                    <tr>
+                        <td>Time calculating diff</td>
+                        <td>{{ object.diff_time | shorter_timedelta }}</td>
+                    </tr>
+                    <tr>
+                        <td>Time enforcing sync</td>
+                        <td>{{ object.sync_time | shorter_timedelta }}</td>
+                    </tr>
+                    {% if object.memory_usage %}
+                    <tr>
+                        <td>Memory usage</td>
+                        <td>{{ object.memory_usage }} bytes</td>
+                    </tr>
+                    <tr>
+                        <td>Memory blocks</td>
+                        <td>{{ object.memory_size }} blocks</td>
+                    </tr>
+                    <tr>
+                        <td>Memory peak blocks</td>
```

Comment:
From a user perspective I don't find these three items very clear. Are `Memory Usage` and `Memory Blocks` measuring the same thing, just in bytes versus blocks? If those two are *not* `Peak` usage, what are they? Can we change any or all of these headings to be clearer in their meaning?

Reply:
I agree with you, this must be improved. I will review it and propose a better implementation of the idea, maybe changing a bit how this is tracked.